src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -136,16 +136,16 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
                     LibraryTab.DatabaseNotification notificationPane = frame.getCurrentLibraryTab().getNotificationPane();
                     notificationPane.notify(
                             IconTheme.JabRefIcons.FOLDER.getGraphicNode(),
-                            Localization.lang("%0 export successful. Open the folder containing the saved file?", format.getName()),
-                            List.of(new Action(Localization.lang("Open"), event -> {
+                            Localization.lang("Export operation finished successfully."),
+                            List.of(new Action(Localization.lang("Reveal in File Explorer"), event -> {
                                 try {
                                     JabRefDesktop.openFolderAndSelectFile(file, preferences);
                                 } catch (IOException e) {
                                     LOGGER.error("Could not open export folder.", e);
                                 }
                                 notificationPane.hide();
                             })),
-                            Duration.seconds(5));
+                            Duration.seconds(10));
```

Comment:
For the export I would not increase the time, 5 seconds  is enough

Reply:
Done